src/model/uniprotkb/sections/ProteinProcessingConverter.ts

```diff
@@ -37,7 +46,14 @@ export const convertProteinProcessing = (data: ProteinProcessingAPIModel) => {
   const proteinProcessingData: ProteinProcessingUIModel = {
     featuresData: [],
     keywordData: [],
+    xrefData: [],
+    commentsData: [],
   };
+  if (data.comments) {
+    proteinProcessingData.commentsData = data.comments.filter(
+      d => d.commentType === Comment.PTM
```

Comment:
can we make a var at the top of the file to hold valid comment types (`Comment.PTM`)